src/convertToObject.js

```diff
@@ -3,16 +3,25 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssClasses = sourceString.split(';')
+    .map(property => property.split(':')
+      .filter(existingProp => existingProp.length > 1)
+      .map(correctProp => correctProp.trim()))
```

Comment:
it's hard to read this code, I've got confused a bit by it, let's fix it:
```suggestion
    .map(property => (
      property
        .split(':')
        .filter(existingProp => existingProp.length > 1)
        .map(correctProp => correctProp.trim())
    )
```